sdk/eventhub/azure-eventhub/migration_guide.md

```diff
@@ -1,14 +1,17 @@
-# Migration Guide (EventHubs v1 to v5)
+# Guide to migrate from azure-eventhub v1 to v5
 
 This document is intended for users that are familiar with V1 of the Python SDK for Event Hubs library (`azure-eventhub 1.x.x`) and wish 
 to migrate their application to V5 of the same library.
 
 For users new to the Python SDK for Event Hubs, please see the [readme file for the azure-eventhub](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub/README.md).
 
 ## General changes
+Version 5 of the azure-eventhub package is a result of our efforts to create a client library that is user-friendly and idiomatic to the Python ecosystem.
+Apart from redesigns resulting from the new [Azure SDK Design Guidelines for Python](https://azure.github.io/azure-sdk/python_introduction.html#design-principles), 
```

Comment:
"Alongside API redesigns driven by the new Azure SDK Design Guidelines, the latest version...."

Reply:
Updated